build/nested-cluster/util/version.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/version"
+)
+
+func main() {
+	// All this does is print the version of k8s we've built.
```

Comment:
as I've noted elsewhere, I don't think you need this program at all.

Reply:
That's true, but I want _something_ in Go, so I have a place to start from.

Reply:
do you intend for this program to do anything besides print out the git version?

Reply:
I want to move the setup/testing into Go.